starboard/linux/shared/cobalt/configuration.py

```diff
@@ -60,6 +60,16 @@
         'HistogramTesterTest.TestGetAllSamples_NoSamples',
         'HistogramTesterTest.TestGetTotalCountsForPrefix',
         'HistogramTesterTest.TestGetAllChangedHistograms',
+
+        # b/316404107
+        # These have flaky ASAN heap-use-after-free during metrics collection.
+        'All/SequenceManagerTest.DelayedTasksDontBadlyStarveNonDelayedWork_DifferentQueue/WithMessagePump',  # pylint: disable=line-too-long
```

Comment:
Is `All/SequenceManagerTest.DelayedTasksDontBadlyStarveNonDelayedWork*` too broad of a filter?

Reply:
IDK which variants all exist. Other variants did not yet trigger a flaky fail, so if they exist, they should continue to run.